### DISCUSSION TITLE
OVH Server no access

### DISCUSSION THREAD

**Discussion by Sufyan6Zahid** [ovh/manager#5546] [Q&A] [answered]
Hi Experts 
OVH sever not access from last one month

**Comment by antleblanc** [comment, marked as answer]
Hi @Sufyan6Zahid!

Unfortunately there's nothing we can do regarding your services from here.
Please contact our support team who will be able to help you or refer to https://help.ovhcloud.com.

Thank you!

**Comment by CRduweb** [comment]
Bonjour, je rencontre un grand problème sur OVH…

Je trouve que l'interface est très complexe à mon niveau !

Mon nom de domaine est actuellement hébergé chez OVH et je dois m'adonner à des configurations tel que relier mon nom de domaine a Google search console, car je suis dans des activités de concepteur de site.

Pouvez-vous m'accorder du temps pour me donner de l'aide ?

Merci pour votre compréhension
